spec/dummy/db/schema.rb

```diff
@@ -400,6 +400,8 @@
     t.bigint "organization_id"
     t.datetime "created_at", null: false
     t.datetime "updated_at", null: false
+    t.integer "subject"
+    t.text "content"
```

Comment:
If we use one of the latter terms, here we could use `custom_content`, `announcement_content` or `custom_announcement_content`, so that it is clear that this field is thighly coupled to the aforementioned subject type